keyboards/nt_engineering/macropad/info.json

```
@@ -0,0 +1,73 @@
+{
+    "keyboard_name": "Macro Pad",
+    "manufacturer": "NT Engineering",
+    "maintainer": "nick-taormina",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "cols": ["GP7", "GP0", "GP1"],
+        "rows": ["GP27", "GP28", "GP29", "GP6"]
+    },
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
```

This is preferred to be left enabled if possible.
```suggestion
        "mousekey": true,
```